exp/controllers/azuremanagedmachinepool_reconciler.go

```diff
@@ -128,3 +149,9 @@ func (r *azureManagedMachinePoolReconciler) Delete(ctx context.Context, scope *s
 
 	return nil
 }
+
+// IsAgentPoolVMSSNotFoundError returns true if the error is a AgentPoolVMSSNotFoundError
+func IsAgentPoolVMSSNotFoundError(err error) bool {
+	_, ok := err.(*AgentPoolVMSSNotFoundError)
```

Comment:
go1.13 errors.Is?

Reply:
You are right. I think I was just being lazy.

Reply:
Fixed. Implemented `Is`.